hack/install.sh

```diff
@@ -119,6 +129,24 @@ do_install() {
 	esac
 
 	if command_exists docker; then
+		version="$(docker -v | awk '{ print $3 }' | sed 's/\,//g')"
```

Comment:
Add `sub(/,$/, "", $3);` before `print` in your `awk` instruction and you can drop `sed` here too, or just use `-F '[ ,]+'` so that comma becomes a separator, ala: `docker -v | awk -F '[ ,]+' '{ print $3 }'`